nautobot_circuit_maintenance/migrations/0007_raw_binary.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.1.13 on 2021-08-09 15:16
+
+from django.db import migrations, models
+
+
+def convert_raw_from_str_to_bytes(apps, schema_editor):
+    """Convert raw from str to bytes."""
+    RawNotificationModel = apps.get_model("nautobot_circuit_maintenance", "RawNotification")
+
+    for raw_notification in RawNotificationModel.objects.all():
+        raw_txt = raw_notification.raw
+        raw_txt = raw_txt.strip("b''").strip("''")
+        raw_notification.raw_b = raw_txt.encode()
+        raw_notification.save(update_fields=["raw_b"])
```

Comment:
So if I understand this right, we essentially have a bug in the previous version in that we were storing a quoted string representation of the raw bytes in a TextField, and now we're trying to migrate to actually storing the raw bytes correctly in a BinaryField? I'm pretty sure that stripping the leading `b'` and trailing `'` and then encoding it to utf-8 are *not* going to give the desired results. If you have a local instance stood up with some of the bad data, I'm happy to jump into nbshell and see if I can help figure out how we should address this - in the worst case, we're early enough yet that it might be easiest to simply throw away bad raw data as part of this migration.

Reply:
yes, I was also considering the option of throw away bad raw data... but I will give an extra try to this conversion and if not I would ping you